src/data/contributors/heenaSingh.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Heena Singh",
+    "skills": ["JavaScipt", "graphQL", "Python", ""],
+    "cause": "Code to make society a better place",
```

Comment:
"code for " is part of the pre-string, you just have to mention the cause.
![image](https://user-images.githubusercontent.com/48255244/94997087-759fc300-05c6-11eb-89bc-e0f823300272.png)